gossip/src/cluster_info.rs

```diff
@@ -271,7 +271,7 @@ pub fn make_accounts_hashes_message(
 pub(crate) type Ping = ping_pong::Ping<[u8; GOSSIP_PING_TOKEN_SIZE]>;
 
 // TODO These messages should go through the gpu pipeline for spam filtering
-#[frozen_abi(digest = "4jtxvWyeFwfDQTTGh4yJLyukALzRNVJ9WNnCbFeJUmaS")]
+#[frozen_abi(digest = "6T2sn92PMrTijsgncH3bBZL4K5GUowb442cCw4y4DuwV")]
```

Comment:
I wonder what cause the ABI change?

Reply:
The digest changes because the struct field name in `LegacyContactInfo` has changed. But that is totally fine here because this structs are only (de)serialized using `bincode` which is totally oblivious to struct field names.